test/test_all.py

```diff
@@ -90,9 +90,9 @@
                     "name": "",
                     "official": False,
                     "tag": "",
-                    "error": "example.com is not currently supported as a Git hosting website for Library Manager.%0A"
-                    "See: https://github.com/arduino/library-registry/blob/main/FAQ.md#what-are-the-requirements-for-a-"
-                    "library-to-be-added-to-library-manager",
+                    "error": "`example.com` is not currently supported as a Git hosting website for Library Manager.%0A"
+                    "%0ASee: https://github.com/arduino/library-registry/blob/main/FAQ.md#what-are-the-requirements-for"
```

Comment:
What's the need of this escape sequence in this case? :thinking:

Reply:
The reason is that this text is used in a GitHub Actions workflow job output. In that application, any text following a standard newline is discarded. So in order to get line breaks it's necessary to use URL-encoding:
https://github.community/t/set-output-truncates-multiline-strings/16852

The alternative is to do the substitution in the workflow, for example:
https://github.com/arduino/arduino-ide/blob/2.0.0-beta.9/.github/workflows/build.yml#L151
but since this output is only ever intended to be used by the workflow, it seems cleaner to encode it correctly from the start.

Reply:
Makes sense. :+1:

Reply:
Since this is not at all obvious, I added an explanatory comment to the first usage of `%0A` in the program.